### DISCUSSION TITLE
Mull: HTTPS-Only no option nor enabled by default

### DISCUSSION THREAD

**Discussion by Icon125** [Divested-Mobile/DivestOS-Build#72] [General] [answered]
Hi, HTTPS-Only mode should be enabled by default in Mull. HTTPS Everywhere is no longer being maintained.

**Comment by SkewedZeppelin** [comment]
https://gitlab.com/divested-mobile/mull-fenix/-/issues/18

**Comment by SkewedZeppelin** [comment]
There does seem to be progress upstream:
https://github.com/mozilla-mobile/fenix/pull/24095

**Comment by SkewedZeppelin** [comment, marked as answer]
HTTPS is now default in Mull 100.1.1
